test_entropy.py

```diff
@@ -20,71 +20,71 @@
-
-    def test_entropy_to_mnemonic(self):
-        e = bipentropy.BIPEntropy()
-        xprv = 'xprv9s21ZrQH143K2LBWUUQRFXhucrQqBpKdRRxNVq2zBqsx8HVqFk2uYo8kmbaLLHRdqtQpUm98uKfu3vca1LqdGhUtyoFnCNkfmXRyPXLjbKb'
-        entropy = e.bip32_xprv_to_entropy("m/83696968'/0'/0'", xprv)
-
-        words12 = 'useful guitar veteran zone perfect october explain grant clarify december flight recycle'
-        words15 = 'useful guitar veteran zone perfect october explain grant clarify december flight raw banana estate uncle'
-        words24 = 'useful guitar veteran zone perfect october explain grant clarify december flight raw banana estate unfair grow search witness echo market primary alley forward boring'
-
-        self.assertEqual(e.entropy_to_bip39(entropy, 12), words12)
-        self.assertEqual(e.entropy_to_bip39(entropy, 15), words15)
-        self.assertEqual(e.entropy_to_bip39(entropy, 24), words24)
-
-    def test_wif_from_entropy(self):
-        e = bipentropy.BIPEntropy()
-        xprv = 'xprv9s21ZrQH143K2LBWUUQRFXhucrQqBpKdRRxNVq2zBqsx8HVqFk2uYo8kmbaLLHRdqtQpUm98uKfu3vca1LqdGhUtyoFnCNkfmXRyPXLjbKb'
-        entropy = e.bip32_xprv_to_entropy("m/83696968'/0'/0'", xprv)
-        self.assertEqual(e.entropy_to_wif(entropy), 'L5G6UFMvJaFt1KPvupEtT8TUN2YrFnQJm1LA2nEczWrR7MuoxB1Z')
-
-    def test_applications(self):
-        e = bipentropy.BIPEntropy()
-        xprv = 'xprv9s21ZrQH143K2LBWUUQRFXhucrQqBpKdRRxNVq2zBqsx8HVqFk2uYo8kmbaLLHRdqtQpUm98uKfu3vca1LqdGhUtyoFnCNkfmXRyPXLjbKb'
-        entropy = e.bip32_xprv_to_entropy("m/83696968'/39'/0'/12'/0'", xprv)
-        self.assertEqual(entropy[:16].hex(), '6250b68daf746d12a24d58b4787a714b')
-        self.assertEqual(e.entropy_to_bip39(entropy, 12),
-                         'girl mad pet galaxy egg matter matrix prison refuse sense ordinary nose')
-
-        entropy = e.bip32_xprv_to_entropy("m/83696968'/39'/0'/18'/0'", xprv)
-        self.assertEqual(entropy[:24].hex(), '938033ed8b12698449d4bbca3c853c66b293ea1b1ce9d9dc')
-        self.assertEqual(e.entropy_to_bip39(entropy, 18),
-                         'near account window bike charge season chef number sketch tomorrow excuse sniff circle vital hockey outdoor supply token')
-
-        entropy = e.bip32_xprv_to_entropy("m/83696968'/39'/0'/24'/0'", xprv)
-        self.assertEqual(entropy[:32].hex(), 'ae131e2312cdc61331542efe0d1077bac5ea803adf24b313a4f0e48e9c51f37f')
-        self.assertEqual(e.entropy_to_bip39(entropy, 24),
-                         'puppy ocean match cereal symbol another shed magic wrap hammer bulb intact gadget divorce twin tonight reason outdoor destroy simple truth cigar social volcano')
-
-
-def __main__():
-    unittest.main()
-
+import pytest
+
+XPRV = 'xprv9s21ZrQH143K2LBWUUQRFXhucrQqBpKdRRxNVq2zBqsx8HVqFk2uYo8kmbaLLHRdqtQpUm98uKfu3vca1LqdGhUtyoFnCNkfmXRyPXLjbKb'
+
+def test_mnemonic():
+    e = bipentropy.BIPEntropy()
+    mnemonic = 'install scatter logic circle pencil average fall shoe quantum disease suspect usage'
```

Comment:
Need to revert the test cases because hmac param order is flipped incorrectly elsewhere in this PR [here](https://github.com/ethankosakovsky/bipentropy/pull/1/commits/a338c764be0e2c084ea345302420cd29aa05ff91#diff-0cdeb850d6883c87a3a53346b43f89faR40)